.github/workflows/abdullahCode.py

```diff
@@ -0,0 +1,225 @@
+print("Hello this is a program that does math for you, press enter after every answer") 
+
+print("")
+
+mathquestion = input("This calculater can do geomatry, quadratic formula, normal calculator,Pythagorean Theorem and more, enter what you want to calculate")
+normalcalculator = input("For normal calculator type 'calculator' or press enter to continue")  
+ # divition, subtraction, addition, multiplication 
+
+if normalcalculator == "calculator":    
+  calculater = input("Enter any expression (multiplication is *): ")  
+  print(" ")
+  print("= ",eval(calculater) ) 
+
+
+def sum(diviona, additionb): 
+    return (divitiona / additionb) 
+if mathquestion == "Divition":
+  divitiona = int(input('Enter 1st number: ')) 
+  additionb = int(input('Enter 2nd number: '))
+  print(f'{divitiona} divided by {additionb} = {sum(divitiona, additionb)}') 
+  
+
+def sum(subtractiona, subtractionb): 
+    return (subtractiona - subtractionb) 
+if mathquestion == "subtraction": 
+  subtractiona = int(input('Enter 1st number: '))
+  subtractionb = int(input('Enter 2nd number: '))
+  print(f'{subtractiona} - {subtractionb} = {sum(subtractiona, subtractionb)}') 
+
+
+def sum(additiona, additionb): 
+    return (additiona + additionb) 
+if mathquestion == "addition":
+  additiona = int(input('Enter 1st number: '))
+  additionb = int(input('Enter 2nd number: '))
+  print(f'{additiona} + {additionb} = {sum(additiona, additionb)}')  
+    
+def sum(multiplicationa, multiplicationb): 
+    return (multiplicationa * multiplicationb) 
+if mathquestion == "multiplication": 
+  multiplicationa = int(input('Enter 1st number: '))
+  multiplicationb = int(input('Enter 2nd number: '))
+  print(f'{multiplicationa} * {multiplicationb} is {sum(multiplicationa, multiplicationb)}') 
+
+# volume of sphere, cube, rectanguler prism, pyramid, cone
+
+import math
+math.pi
+pi = math.pi 
+if mathquestion == "volume": 
+ volumeshape = input("What shape? ") 
+ if volumeshape == "sphere":
+   raduispherevolume = int(input("What's the raduis? "))  
+   print(4*math.pi*raduispherevolume**3/3) 
+
+
+if mathquestion == "volume":
+ if volumeshape == "cube": 
+   sidecubevolume = int(input("What is the side length? "))  
+   print(sidecubevolume**2) 
+
+if mathquestion == "volume":
+ if volumeshape == "rectangulrer prism": 
+   rectangulerprismvolumel = int(input("what is the length? ")) 
+   rectangulerprismvolumew = int(input("What is the width? ")) 
+   rectangulerprismvolumeh = int(input("What is the height? " )) 
+   print(rectangulerprismvolumel * rectangulerprismvolumew * rectangulerprismvolumeh) 
```

Comment:
print a message as well lik 
volume of rectangular prism is : 
also its hard for user to type exact name, 
you should take option in the form of number 
1. Addition
2. Multiplication
3. Volume ETC
4.